integration-tests/csimple/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name" : "java.lang.String",
```

Comment:
Nitpick - we could do this with `@RegisterForReflection`.

Reply:
Nice one, thx for sharing 🙏